### ISSUE TITLE
How to showCallout() when a marker is initially added to the map

### ISSUE THREAD

**Issue by kcfgl** [react-community/react-native-maps#1647]
I checked SO first because this isn't an issue, but I couldn't find anything.  What I'm trying to do is pretty straightforward. 

When I long press on a map, it adds a new Marker with a custom callout.  Tapping the marker shows the callout as expected.  

The only change is I'd like to be able to have the callout visible right after the marker is added to the map.

Thanks!

**Comment by kytwb** [comment]
You can manually call `showCallout` on the marker at your convenience.

**Comment by kcfgl** [comment]
Thanks. I definitely know that is available. But I’m more interested in the proper way to handle this or a best practice to ensure rendering is efficient. If I do it manually, I have to call hideCalllout too, whereas if I tap to show, then I automatically get default behaviors like tapping the map auto hides the callout. 

If I add a pin to the map, rendering is going to happen, so I have looked at adding a flag to show/hide callout. But that isn’t working well. 

So it was more of a question for anyone who successfully implemented this type of use case and how they handled it for maximum efficiency. 

Or the way to do this so the pin still  functions exactly as it would if I tapped the pin. Thanks!
